dist/images/ovnkube.sh

```diff
@@ -524,12 +524,27 @@ ovn-northd () {
   fi
 }
 
+# make sure ports 6641/6642 are open on master
+iptables-rules () {
+iptables -C INPUT -p tcp -m tcp --dport 6642 -m conntrack --ctstate NEW -j ACCEPT
+if [[ $? != 0 ]] ; then
+  iptables -A INPUT -p tcp -m tcp --dport 6642 -m conntrack --ctstate NEW -j ACCEPT
+fi
+iptables -C INPUT -p tcp -m tcp --dport 6641 -m conntrack --ctstate NEW -j ACCEPT
+if [[ $? != 0 ]] ; then
+  iptables -A INPUT -p tcp -m tcp --dport 6641 -m conntrack --ctstate NEW -j ACCEPT
+fi
+
+}
+
```

Comment:
@pecameron The only thing I would suggest is, if possible, to add a comment to the rules we are adding

`iptables -m comment --comment "OVN Northbound DB  port"` and `iptables -m comment --comment "OVN Southbound DB port"`.

Reply:
The check and delete match the whole rule including comment. I prefer not to add more than necessary to the rule.

Reply:
For check, you don't have to also include the comment part. You will just need to check for protocol and port itself. It helps others who don't know why TCP ports 6641 and 6642 are being opened on the master node.  Also, we already add a comment to other rules added by ovnkube.

`ACCEPT     all  --  anywhere             anywhere             /* from OVN to localhost */`

Reply:
The check does require the comment to match, at least on my test system. Try it out.

Reply:
@pecameron I see. If so, then it is fine.